app/src/main/java/com/android/surveysaurus/activity/MainActivity.kt

```diff
@@ -0,0 +1,123 @@
+package com.android.surveysaurus.activity
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.ActionMenuView
+import androidx.core.view.isVisible
+import androidx.navigation.findNavController
+import com.android.surveysaurus.singleton.LoginSingleton
+import com.android.surveysaurus.R
+import com.android.surveysaurus.databinding.ActivityMainBinding
+
+
+class MainActivity : AppCompatActivity() {
+private  lateinit var profileActionMenuView: ActionMenuView
```

Comment:
genel olarak kod formatlamalarında sıkıntılar var. Dosyaları formatlayıp gönderelim. Örneğin bu satır diğerlerinin gerisinde

Reply:
Formatlama yapıldı.